src/api/routes/node.ts

```diff
@@ -31,6 +31,12 @@ export const NodeRoutes: FastifyPluginCallback<Record<never, never>, Server, Typ
         contract: Type.String({examples: ['Marbling']}),
         var: Type.String({examples: ['base-uri']}),
       }),
+      response: {
+        200: Type.String({
```

Comment:
This should also be an Object (same below)

Reply:
@rafaelcr as i explained in another PR, because of the return type is defined as `reply.type('application/json').send(JSON.stringify({` when i have the return type defined as Type.Object, compile fails for this. 

I can change this to `Type.Object` if the code is changed too. Or would want me to modify both and submit a PR?

Reply:
Ahh you're right... I think we did that to format the output, let me change the responses quickly and get back to you

Reply:
Thanks!

Reply:
Done @LakshmiLavanyaKasturi you can now merge and use objects 👍

Reply:
@rafaelcr: Pushed response as objects, please review.